include/cocaine/framework/worker/options.hpp

```diff
@@ -41,6 +41,8 @@ struct options_t {
     std::uint32_t
     protocol() const;
 
+    const boost::any&
+    at(const std::string& name) const;
```

Comment:
Closer, but not what I wanted.

Look, we have predefined number of options passed from cocaine-runtime. Therefore we can describe them explicitly:

```c++
std::string token_type() const;
std::string token_body() const;
```